### PR TITLE
2.14 version OHOS adaptation

### PR DESCRIPTION
**Pull request by IceBBBBB**

### CODE REVIEW

File: cocos2dx/textures/CCTexture2D.cpp [hunk 1]
```diff
@@ -511,12 +512,17 @@ bool CCTexture2D::initWithString(const char *text, const char *fontName, float f
...
     #if CC_ENABLE_CACHE_TEXTURE_DATA
         // cache the texture data
-        VolatileTexture::addStringTexture(this, text, dimensions, hAlignment, vAlignment, fontName, fontSize);
-    #endif
+		#if(CC_TARGET_PLATFORM == CC_PLATFORM_OHOS)
+            VolatileTexture::addStringTexture(this, text, textDefinition->m_dimensions, textDefinition->m_alignment, textDefinition->m_vertAlignment, textDefinition->m_fontName.c_str(), textDefinition->m_fontSize);
+    	#else
+		    VolatileTexture::addStringTexture(this, text, dimensions, hAlignment, vAlignment, fontName, fontSize);
+		#endif
+	#endif
```
Comment (minggo) [inline cocos2dx/textures/CCTexture2D.cpp]: Wrong indent.

**Comment by minggo** [comment]
Third libraries are included in `platform/third_party`. And it is better to use prebuilt library.

File: cocos2dx/textures/CCTexture2D.cpp (hunk 1 above)
Comment (IceBBBBB) [reply cocos2dx/textures/CCTexture2D.cpp]: modified

**Comment by IceBBBBB** [comment]
> Third libraries are included in `platform/third_party`. And it is better to use prebuilt library.

Version 2.1.4 is relatively old and difficult to compile, with higher versions using prebuilt library

**Comment by minggo** [comment]
Ok, then please add them in platform/third_party.

**Comment by IceBBBBB** [comment]
> Ok, then please add them in platform/third_party.

modified.

**Pull request merged**